src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -497,11 +496,23 @@ ov::AnyMap get_baseline_common_config() {
         { "NPUW_SLICE_OUT", "YES" },
         { "NPUW_FUNCALL_ASYNC", "YES" }
     };
+    if (enable_compiler_dq) {
+        config.emplace("NPUW_DQ", "YES");
+        config.emplace("NPUW_DQ_FULL", "NO");
+        config.emplace("NPU_COMPILER_DYNAMIC_QUANTIZATION", true);
+        config.erase("NPUW_DCOFF_TYPE");
+        config.erase("NPUW_DCOFF_SCALE");
+    }
     return config;
 }
 
-ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
-    auto config = get_baseline_common_config();
+bool enable_compiler_dq(const std::optional<NPUDesc>& npudesc) {
```

Comment:
```suggestion
bool have_compiler_dq(const std::optional<NPUDesc>& npudesc) {
```
`enable_` may look ambiguous depending on the context (as it already does in the code below)

Reply:
Removed this function